wallets/src/main/java/bisq/wallets/elementsd/process/ElementsdProcess.java

```diff
@@ -49,7 +48,7 @@ public ProcessConfig createProcessConfig() {
         return new ProcessConfig(
                 "elementsd",
                 List.of(
-                        "-chain=" + getNetworkName(),
+                        "-chain=elementsregtest",
```

Comment:
Is that correct?

Reply:
Yes, that's correct but you're right. It's confusing. I renamed it to `ElementsdRegtestProcess` and moved it into the tests module.